services/web-server/src/loaders/tasks.js

```diff
@@ -53,7 +53,7 @@ module.exports = ({ queue, index }, isAuthed, rootUrl, monitor, strategies, req,
             'public/actions.json',
           );
 
-          const raw = await fetch(url);
+          const raw = await got(url).json();
```

Comment:
`got` supports [retries](https://github.com/sindresorhus/got/blob/main/documentation/7-retry.md#retry) by default, so no need for custom fetch implementation